recipes/pear/meta.yaml

```diff
@@ -1,22 +1,28 @@
 package:
   name: pear
   version: '0.9.6'
+
 about:
   home: http://sco.h-its.org/exelixis/web/software/pear/
   license: 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported'
   summary: 'paired-end read merger'
+
 source:
   fn: pear-0.9.6-src.tar.gz
   md5: 5ed68d50c1620cd55c58681e38771799
   url: http://sco.h-its.org/exelixis/web/software/pear/files/pear-0.9.6-src.tar.gz
+
 build:
-  number: 1
+  number: 2
+
 requirements:
   build:
+    - gcc
     - pkgconfig
     - zlib
   run:
     - zlib
```

Comment:
I’ll add it. But [the error](https://travis-ci.org/bioconda/bioconda-recipes/jobs/170718680#L599) occurs at build time. May have something [to do with zlib](https://travis-ci.org/bioconda/bioconda-recipes/jobs/170718680#L577).